packages/venia-ui/lib/components/FilterModal/CurrentFilters/currentFilter.js

```diff
@@ -9,15 +10,26 @@ import defaultClasses from './currentFilter.css';
 
 const CurrentFilter = props => {
     const { group, item, removeItem } = props;
+    const { formatMessage } = useIntl();
     const classes = mergeClasses(defaultClasses, props.classes);
 
     const handleClick = useCallback(() => {
         removeItem({ group, item });
     }, [group, item, removeItem]);
 
+    const ariaLabel = formatMessage(
+        {
+            id: 'filterModal.action.clearFilterItem.ariaLabel',
```

Comment:
Can you add this to [en_US.json](https://github.com/magento/pwa-studio/blob/develop/packages/venia-ui/i18n/en_US.json) please :)